test/irb/test_debugger_integration.rb

```diff
@@ -434,5 +434,25 @@ def test_multi_irb_commands_are_not_available_after_activating_the_debugger
       assert_match(/irb\(main\):001> next/, output)
       assert_include(output, "Multi-IRB commands are not available when the debugger is enabled.")
     end
+
+    def test_irb_passes_empty_input_to_debugger_to_repeat_the_last_command
+      write_ruby <<~'ruby'
+        binding.irb
+        puts "foo"
+        puts "bar"
+        puts "baz"
+      ruby
+
+      output = run_ruby_file do
+        type "next"
+        type ""
+        type "\n\r"
+        type "\n"
```

Comment:
`type "\n"` is exactly same as `type ""` because `puts("")` is equal to `puts("\n")`
```ruby
# type(command) pushes command string to `@commands`
# method run_ruby_file in test/irb/helper.rb L129-131
while command = @commands.shift
  write.puts(command)
end
```

`type "\n\r"` writes `"\n\r\n"`. `\r` and `\n` are both assigned to `ed_newline` in `reline/key_actor/emacs.rb`. `type("\n\n\n")` is equivalent to `3.times{type ""}`.

I think we can use `type "\e\n\n"`. (`"\e\n"` is option+enter)

Reply:
The idea was to test different ways of sending empty input to make sure we've covered them all with the implementation. But it looks like it's not necessary after all 😅  How about we test it simply with `""`?

Reply:
> How about we test it simply with `""`?

I think it's ok 👍 

We can think of a way to pass multiline input in IntegrationTestCase in the future.